mir-ci/mir_ci/wayland/virtual_pointer.py

```diff
@@ -53,23 +54,26 @@ def disconnected(self) -> None:
         pass
 
     def xdg_output_logical_size(self, xdg_output, width: int, height: int) -> None:
+        # Whenever the logical size changes, we updated the cached physical size
         if xdg_output == self.xdg_outputs[0]:
-            self.output_width = width
-            self.output_height = height
+            self.output_physical_width = int(width * self.output_scale)
+            self.output_physical_height = int(height * self.output_scale)
```

Comment:
I think I would rather stick to `output_width/height`, the only place where we have non-physical geometry is `xdg_output_logical_size`, so let's contain it there.